doc/source/ray-core/handling-dependencies.rst

```diff
@@ -405,7 +407,7 @@ The ``runtime_env`` is a Python dictionary or a Python class :class:`ray.runtime
   - Example: ``"pytorch_p36"``
 
   When specifying a path to a ``environment.yml`` file, the file must be present on your local machine and it must be a valid absolute path or a relative filepath relative to your local current working directory, *not* relative to the `working_dir` specified in the `runtime_env`.
-  Furthermore, referencing local files `within` a `environment.yml` file is not supported.
+  Furthermore, referencing local files *within* a `environment.yml` file is not directly supported. Instead, a environment variable `${RAY_RUNTIME_ENV_CREATE_WORKING_DIR}` is provided in the creation process.
```

Comment:
```suggestion
  Furthermore, referencing local files *within* a `environment.yml` file isn't directly supported. Instead, use the `${RAY_RUNTIME_ENV_CREATE_WORKING_DIR}`environment variable in the creation process.
```

Reply:
Nit: Let's also include your full description from above, it's pretty helpful.

> Instead, use the `${RAY_RUNTIME_ENV_CREATE_WORKING_DIR}` environment variable in the creation process. For example, use `-r ${RAY_RUNTIME_ENV_CREATE_WORKING_DIR}/my-laptop/more-requirements.txt` or `${RAY_RUNTIME_ENV_CREATE_WORKING_DIR}/my-pkg.whl` to reference local files, while ensuring they're in the `working_dir`.

(Or whatever is required to make it work for `conda`.  It's just helpful to have a more fleshed out example.)